.github/workflows/release-drafter.yml

```diff
@@ -2,15 +2,20 @@ name: Release Drafter
 
 on:
   push:
-    branches: [main]
+    tags:
+      - 'v*'
 
 jobs:
   update_release_draft:
+    if: github.event.base_ref == 'refs/heads/main'
     runs-on: ubuntu-latest
     steps:
-      # Drafts your next Release notes as Pull Requests are merged into "main"
+      # Publishes new Release as new git tag (vX.Y.Z) is pushed to repository
       - uses: release-drafter/release-drafter@v5
         with:
           config-name: release-drafter-config.yml
+          publish: true
```

Comment:
Pls show an example of how such release looks like from your private repo

Reply:
![image](https://user-images.githubusercontent.com/49597713/170456615-4a3779de-3aa2-4f80-917d-018e6532968d.png)